Task_8/style.css

```diff
@@ -0,0 +1,373 @@
+    margin: 50px auto;
+    padding: 7px;
+    height: 320px;
+    width: 320px;
+    background-color: #c7c7c7;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    align-items: center;
+}
+.res_x{
+    margin:auto;
+    height: 320px;
+    width: 320px;
+    background-color: #362f63;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    color: #c7c7c7;
+    font-size: 50px;
+    display: none;
+}
+.res_x:hover{
+    cursor: no-drop;
+}
+.res_0{
+    margin:auto;
+    height: 320px;
+    width: 320px;
+    background-color: #632f3b;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    color: #c7c7c7;
+    font-size: 50px;
+    display: none;
+}
+.res_0:hover {
+    cursor: no-drop;
+}
+/*Изменение игровой ячейки*/
+.game_field:hover{
+    transition: 0.3s;
+    box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    -webkit-box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    -moz-box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    background-color: #dba587;
+}
+/*Игровая ячейка*/
+.game_field{
+    display: flex;
+    width: 100px;
+    height: 100px;
+    background-color: #e8ae8f;
+    border-radius: 4px;
+    justify-content: center;
+    align-items: center;
+}
+.ico_big{
+    width: 90px;
+    height: 95px;
+    padding: 4px;
+}
+.big{
+    width: 100px;
+    height: 100px;
+    display: flex;
+    background-color: #e8ae8f;
+    justify-content: center;
+    align-items: center;
+    display: none;
+    cursor: pointer;
+}
+.big:hover{
+    transition: 0.3s;
+    box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    -webkit-box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    -moz-box-shadow:0px 0px 9px 1px rgba(102,41,0,1);
+    background-color: #dba587;
+}
+a{
+    margin: 20px auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #424242;
+    text-decoration: none;
+    background-color: #89a73b;
+    font-size: 20px;
+    width: 180px;
+    height: 60px;
+    border-radius: 7px;
+    border-bottom: 4px solid #6b802d;
+    font-weight: 600;
+    transition: 0.5s;
+}
+a:hover{
+    background-color: #6b802d;
+    transition: 0.5s;
+    color: #181818;
+    font-size: 21px;
+}
+/*Все инпуты скрыты*/
+input{
+    display: none;
+}
+/*.new_game{*/
+/*    text-align: center;*/
+/*    margin: 20px;*/
+/*    background-color: #89a73b;*/
+/*    width: 150px;*/
+/*}*/
+.small{
+    width: 35px;
+    height: 35px;
+    opacity: 0.5;
+    transition: 0.5s;
+}
+.small:hover{
+    cursor: pointer;
+    width: 50px;
+    height: 50px;
+    opacity: 0.8;
+}
+input[name="X1"]:checked ~ #big_1_x{
+    display: block;
+}
+input[name="X1"]:checked ~ .game_field_1{
+    display: none;
+}
+input[name="O1"]:checked ~ #big_1_0{
+    display: block;
+}
+input[name="O1"]:checked ~ .game_field_1{
+    display: none;
+}
+/*___________________________________________*/
+input[name="X2"]:checked ~ #big_2_x{
```

Comment:
I am sure that we can create only one selector for all checked `X` and second selector for all checked `O`
For example:

```
input[name^="X"]:checked + div + [id=^"big_x"] {
display: block;
}
```
instead of a lot of 
`input[name="X2"]:checked ~ #big_2_x {`